scripts/xtensa-build-zephyr.sh

```diff
@@ -196,6 +196,9 @@ build_platforms()
 				;;
 			tgl-h|tgl)
 				PLAT_CONFIG='intel_adsp_cavs25'
+				if test $platform = tgl-h ; then
+				    PLAT_CONFIG="$PLAT_CONFIG@2"
```

Comment:
Please add a one-line comment with the keywords "board revision" because 1. there is no way to guess unless you're pretty familiar with Zephyr already, 2. using a board revision and Kconfig to override (?) and match the number of cores isn't exactly intuitive.